ospfd/ospf_opaque.c

```diff
@@ -2076,35 +2076,33 @@ void ospf_opaque_lsa_refresh_schedule(struct ospf_lsa *lsa0)
 		goto out;
 	}
 
+	ospf = ospf_lookup_by_vrf_id(VRF_DEFAULT);
```

Comment:
can we call this only in else condition?

Reply:
> can we call this only in else condition?

Ok - pushed with this change.